src/pages/workspace/distanceRates/PolicyDistanceRatesPage.tsx

```diff
@@ -0,0 +1,282 @@
+            <Text style={[styles.searchInputStyle, styles.textAlignCenter]}>{translate('statusPage.status')}</Text>
+        </View>
+    );
+
+    const getBulkActionsButtonOptions = () => {
+        const options: Array<DropdownOption<WorkspaceDistanceRatesBulkActionType>> = [
+            {
+                text: translate(`workspace.distanceRates.${selectedDistanceRates.length <= 1 ? 'deleteRate' : 'deleteRates'}`),
+                value: CONST.POLICY.DISTANCE_RATES_BULK_ACTION_TYPES.DELETE,
+                icon: Expensicons.Trashcan,
+                onSelected: deleteRates,
+            },
+        ];
+
+        const enabledRates = selectedDistanceRates.filter((rate) => rate.enabled);
+        if (enabledRates.length > 0) {
+            options.push({
+                text: translate(`workspace.distanceRates.${enabledRates.length <= 1 ? 'disableRate' : 'disableRates'}`),
+                value: CONST.POLICY.DISTANCE_RATES_BULK_ACTION_TYPES.DISABLE,
+                icon: Expensicons.DocumentSlash,
+                onSelected: disableRates,
+            });
+        }
+
+        const disabledRates = selectedDistanceRates.filter((rate) => !rate.enabled);
+        if (disabledRates.length > 0) {
+            options.push({
+                text: translate(`workspace.distanceRates.${disabledRates.length <= 1 ? 'enableRate' : 'enableRates'}`),
+                value: CONST.POLICY.DISTANCE_RATES_BULK_ACTION_TYPES.ENABLE,
+                icon: Expensicons.DocumentSlash,
+                onSelected: enableRates,
+            });
+        }
+
+        return options;
+    };
+
+    const headerButtons = (
+        <View style={[styles.w100, styles.flexRow, isSmallScreenWidth && styles.mb3]}>
+            {selectedDistanceRates.length === 0 ? (
+                <>
+                    <Button
+                        medium
+                        text={translate('workspace.distanceRates.addRate')}
+                        onPress={addRate}
+                        style={[styles.mr3, isSmallScreenWidth && styles.flexGrow1]}
+                        icon={Expensicons.Plus}
+                        iconStyles={[styles.mr2]}
+                        success
+                    />
+
+                    <Button
+                        medium
+                        text={translate('workspace.common.settings')}
+                        onPress={openSettings}
+                        style={[isSmallScreenWidth && styles.flexGrow1]}
+                        icon={Expensicons.Gear}
+                        iconStyles={[styles.mr2]}
+                    />
+                </>
+            ) : (
+                <ButtonWithDropdownMenu<WorkspaceDistanceRatesBulkActionType>
+                    shouldAlwaysShowDropdownMenu
+                    pressOnEnter
+                    customText={`${selectedDistanceRates.length} ${translate('workspace.distanceRates.selected')}`}
```

Comment:
NAB: we can do something like this https://github.com/Expensify/App/blob/7b51c3a568750dcd858de4b92530df60d130da2c/src/languages/en.ts#L1818, it'd also be great if we can reuse it.